packages/protocol/src/session/Session.ts

```diff
@@ -29,7 +29,7 @@ import { SessionParameters } from "./SessionParameters.js";
 
 export class NonOperationalSession extends ImplementationError {
     constructor(session: Session) {
-        super(`Session ${session.name} has no channel assigned`);
+        super(`Session ${session.name} has ended`);
```

Comment:
formally could also have never been initialized ... but yeah common usecase is ended

Reply:
Yeah we could consider changing, it does show up in kind of weird places when there are bugs

Reply:
I consolidated with SessionClosedError so this one has gone away.